packages/shared/src/components/notifications/utils.ts

```diff
@@ -50,3 +51,8 @@ export const notificationDefaultTheme: Record<NotificationIcon, string> = {
   [NotificationIcon.Upvote]: 'text-theme-color-blueCheese',
   [NotificationIcon.Bell]: '',
 };
+
+const notificationsUrl = `${webappUrl}/notifications`;
+
+export const checkAtNotificationsPage = (): boolean =>
```

Comment:
This will not work in the extension

Reply:
That's alright, there is no notifications page in the extension, only in webapp so we will never have to use the comparison.

Reply:
Oh so the bell button opens the notifications page in the webapp always?

Reply:
Yes. Should we create a new page in the extension? Or should I ask product?

Reply:
That's fine. Let's keep it as it is